main.cpp

```diff
@@ -1,3 +1,8 @@
+#include <iostream>
+
+#include <iostream>
```

Comment:
included header iostream is not used directly [misc-include-cleaner,-warnings-as-errors]